source/community/faq/index.md

```diff
@@ -47,6 +51,18 @@ IIIF allows for:
  * [Authentication API][auth]
  * In Progress: [IIIF for Audio/Visual materials][av]
 
+##  How does IIIF relate to my current, internal interoperability layer?
+
+* Some institutions implement IIIF using a shim, or translation layer, on top of their existing interoperability layer.  In other cases, IIIF is implemented without dependencies on existing services. IIIF provides for different levels of compliance, beginning with the IIIF Image API. For more information, please see [IIIF technical details][tech-details].  
+
+## Does IIIF replace or integrate with my Digital Asset Management (DAM) software?
+
+* IIIF is a method for delivery of assets, not storage. Integration with your DAM software is dependent on your DAM provider. The IIIF community is currently in the midst of asking popular DAMS providers to support IIIF.
```

Comment:
I wouldn't put a time dependent statement in a FAQ ... we'll forget to take it out later.  An encouragement to request adoption of vendors could be appropriate though?